app/views/shared/_services-nav.html.erb

```diff
@@ -1,50 +1,108 @@
-
-    <span class="shortcuts__divider">|</span>
-
-    <%= link_to_unless_current "Pending", admin_requests_path, class: "shortcuts__link" do %>
-        <span class="shortcuts__current">Pending</span>
-    <% end %>
-    <span class="shortcuts__count">(<%= @pending_count %>)</span>
-
-    <span class="shortcuts__divider">|</span>
-
-    <% if (params[:archived] === "true") %>
-        <span class="shortcuts__current">Archived</span>
-    <% else %>
-        <%= link_to "Archived", admin_services_path(archived: true), class: "shortcuts__link" %>
-    <% end %>
-
-    <span class="shortcuts__count">(<%= @archived_count %>)</span>
+  <% 
+    @directory_links = {}
+
+    if APP_CONFIG["directories"].present?
+      APP_CONFIG["directories"].each do |directory|
+        @directory_links[directory["value"]] = admin_services_path(directory: directory["value"]);
+      end
+    end
+
+%>
+  
+  <%= 
+  # display the bfis/bod headings
+  render partial: "shared/directory-shortcuts",  locals: { 
+        all_state: !params[:directory].present?,
+        all_link: admin_services_path,
+        all_count: @service_counts[:all][:all],
+        all_label: "All",
+        directory_links: @directory_links,
+        directory_counts: @service_counts
+  }
+
+  #directory_link: admin_services_path(directory: directory["value"])
+%>
+
+
+
+<nav class="shortcuts shortcuts--no-top-spacing  shortcuts--sublevel">
+
+<% 
+
+
+    @sublevel_ofsted_state = false;
+    @sublevel_pending_state = false;
+    @sublevel_archived_state = false;
+
+    # toggle the active state for the ofsted filter
+    if params[:ofsted].present? && params[:ofsted] === "true" 
+        @sublevel_ofsted_state = true;
+    else    
+        @sublevel_ofsted_state = false;
+    end
+
+    # toggle the active state for the pending filter
+    if current_page?(controller: 'admin/requests', action: 'index') 
+        @sublevel_pending_state = true;
+    else    
+        @sublevel_pending_state = false;
+    end
+
+    # toggle the active state for the archived filter
+    if params[:archived].present? && params[:archived] === "true" 
+        @sublevel_archived_state = true;
+    else    
+        @sublevel_archived_state = false;
+    end
+    
+
+    # decide what the urls will be like for the active/deactivated links
+    @sublevel_ofsted_inactive_link = admin_services_path(directory: params[:directory], ofsted: true);
+    @sublevel_pending_inactive_link = admin_requests_path(directory: params[:directory]);
+    @sublevel_archived_inactive_link = admin_services_path(directory: params[:directory], archived: true);
+
+
+    # decide what the counts will be for the active/deactivated links
+    if !params[:directory].present? 
+        @sublevel_ofsted_count = @service_counts[:all][:ofsted];
+        @sublevel_pending_count  = @service_counts[:all][:pending];
+        @sublevel_archived_count  = @service_counts[:all][:archived];
+    elsif
```

Comment:
should just be `else` if no other condition. would probably do the other way round, so:
```
if params[:directory].present?
...
else
...
end